adit/core/errors.py

```diff
@@ -9,6 +9,60 @@ class RetriableDicomError(Exception):
     pass
 
 
+class DicomConversionError(Exception):
```

Comment:
I would here also prefer `DcmToNiftiConversionError` as it's clearer. Also why is this in `core`? It's only used in `dicom_web`!